projects/llvm/build.sh

```diff
@@ -34,7 +68,7 @@ cmake -GNinja -DCMAKE_BUILD_TYPE=Release ../llvm \
     -DLLVM_USE_SANITIZE_COVERAGE=YES \
     -DLLVM_USE_SANITIZER=Address
 
-for fuzzer in clang-fuzzer clang-format-fuzzer llvm-dwarfdump-fuzzer; do
+for fuzzer in clang-format-fuzzer llvm-dwarfdump-fuzzer; do
```

Comment:
please migrate the remaining targets upstream and delete the last part from here